packages/core/src/LottieView.tsx

```diff
@@ -11,6 +16,7 @@ type Props = LottieViewProps & { containerProps?: ViewProps };
 
 const defaultProps: Props = {
   source: undefined,
+  sourceDotLottie: undefined,
```

Comment:
Considering you defined `_parsePossibleSources` below to create the actual source from `source` prop, i think you can get rid of this prop and it's usage. No reason to let people get confused about it